packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts

```diff
@@ -91,18 +96,20 @@ export function getBlockExplorerByBlockchain(chain: EVMBlockchainIncludingTestne
         ["ethereum-goerli", "https://goerli.etherscan.io"],
         ["ethereum-sepolia", "https://sepolia.etherscan.io"],
         ["polygon-mumbai", "https://mumbai.polygonscan.com"],
-        ["zkatana", "https://zkatana.explorer.startale.com"],
-        ["arbitrum-sepolia", null],
-        ["base-goerli", null],
-        ["base-sepolia", null],
-        ["bsc-testnet", null],
-        ["optimism-goerli", null],
-        ["optimism-sepolia", null],
-        ["zora-goerli", null],
-        ["zora-sepolia", null],
-        ["base", null],
-        ["zora", null],
-        ["arbitrumnova", null],
+        ["zkatana", "https://zkatana.blockscout.com"],
+        ["arbitrum-sepolia", "https://sepolia.arbiscan.io"],
+        ["base-goerli", "https://goerli.basescan.org"],
+        ["base-sepolia", "https://sepolia.basescan.org"],
+        ["bsc-testnet", "https://testnet.bscscan.com"],
+        ["optimism-goerli", "https://goerli-optimism.etherscan.io"],
+        ["optimism-sepolia", "https://sepolia-optimism.etherscan.io/"],
+        ["zora-goerli", "https://goerli-optimism.etherscan.io"],
```

Comment:
this says optimism?

Reply:
hmm looks like it may have been https://testnet.explorer.zora.energy/ but is now not. If we do null here that's fine, it's probably no longer in existence